subcommand/server-acl-init/rules.go

```diff
@@ -201,11 +202,20 @@ namespace "{{ .ConsulSyncDestinationNamespace }}" {
 }
 
 func (c *Command) injectRules() (string, error) {
-	// The Connect injector only needs permissions to create namespaces.
+	// The Connect injector needs permissions to create namespaces and also
+	// to register and update service checks.
 	injectRulesTpl := `
 {{- if .EnableNamespaces }}
 operator = "write"
 {{- end }}
+{{- if .EnableHealthChecks }}
```

Comment:
Tests please

Reply:
sorry forgot to push my last commit which removes all of this.